### DISCUSSION TITLE
Spamming echo bar only java-mode

### DISCUSSION THREAD

**Discussion by franklintimoteo** [joaotavora/eglot#1277] [Q&A]
Not sure if this is a bug so asking here first. Typing any character in a java file. The echo bar receives spam from the eglot. 

Reproduction recipe
1-Just activate eglot-mode and type anything.

[eglot] java hashrandom: / Publish Diagnostics 
[eglot] java hashrandom: / Validate Documents
![2023-08-21-19:10:53-screenshot](https://github.com/joaotavora/eglot/assets/11031211/a3d7255e-7737-4333-84d2-0517cf572972)

**Comment by joaotavora** [comment]
You didn't say which version of Eglot you are using.

**Reply by franklintimoteo** [reply]
Hello Joao! 
-- EMACS
extra/emacs 29.1-4 [instalado]

-- EGLOT
     Status: Available from gnu -- Install
    Archive: gnu
    Version: 1.15
     Commit: 30892cbd330ace125454a056ca86d85e2a860bf5
    Summary: The Emacs Client for LSP servers

**Reply by franklintimoteo** [reply]
After to use refresh packages with package-list-packages implicit, and reinstall eglot seems works normaly.
All logs from event buffer log displayed on echo area.
